ci/lib.sh

```diff
@@ -36,7 +36,7 @@ run_static_checks()
 	# Make sure we have the targeting branch
 	git remote set-branches --add origin "${branch}"
 	git fetch -a
-	bash "$tests_repo_dir/.ci/static-checks.sh" "github.com/kata-containers/kata-containers"
+	bash "$tests_repo_dir/.ci/static-checks.sh" $@
```

Comment:
```suggestion
	bash "$tests_repo_dir/.ci/static-checks.sh" "$@"
```
Could you double-quote `$@`?

Reply:
Huh. I was convinced that that would squash all arguments into one, but shell quoting never ceases to ~confuse the heck out of~ amaze me. Updated.